Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM openshift/origin-cli:v3.11
+
+# Environment
+ARG IN_DOCKER_CONTAINER="true"
+ENV VENV=/venv
+ENV REPO_PATH=/managed-cluster-config
+
+# Copy repo into docker image:
+COPY . ${REPO_PATH}
+WORKDIR ${REPO_PATH}
+
+# Docker dependencies:
+RUN yum update -y
+RUN yum install -y git
+RUN yum install -y make
+
+# Makefile dependencies:
+RUN curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
```

Comment:
this method of installing pip is a potential problem. If someday this url is off, our CI breaks as well and we can't merge PRs into that repo.

Reply:
The preferred way is actually not working...
```
[root@a8652c0b01db /]# yum install python-pip python-wheel
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors
 * base: mirrors.mit.edu
 * extras: mirrors.mit.edu
 * updates: mirrors.mit.edu
base                                                                                                                                                                    | 3.6 kB  00:00:00     
cbs-paas7-openshift-multiarch-el7-build                                                                                                                                 | 3.4 kB  00:00:00     
extras                                                                                                                                                                  | 3.4 kB  00:00:00     
updates                                                                                                                                                                 | 3.4 kB  00:00:00     
(1/6): base/7/x86_64/group_gz                                                                                                                                           | 166 kB  00:00:00     
(2/6): extras/7/x86_64/primary_db                                                                                                                                       | 205 kB  00:00:00     
(3/6): updates/7/x86_64/primary_db                                                                                                                                      | 6.5 MB  00:00:00     
(4/6): cbs-paas7-openshift-multiarch-el7-build/x86_64/group_gz                                                                                                          |  458 B  00:00:00     
(5/6): base/7/x86_64/primary_db                                                                                                                                         | 6.0 MB  00:00:00     
(6/6): cbs-paas7-openshift-multiarch-el7-build/x86_64/primary_db                                                                                                        | 6.2 MB  00:00:02     
No package python-pip available.
No package python-wheel available.
Error: Nothing to do
```